packages/react/src/components/Authenticator/SignUp/SignUp.tsx

```diff
@@ -114,12 +114,12 @@ export function SignUp() {
           )}
 
           {secondaryAliases
-            .filter((alias) => !includes(socialProviderLoginMechanisms, alias))
+            .filter((alias) => alias as UserNameAlias)
```

Comment:
I don't think this is going to properly filter.

https://www.typescriptlang.org/play?#code/C4TwDgpgBAggNgSwIYGcoF4oHICuKIBOAdkgLYRZQA+2EpSCclNWYAFgPZEUDcAsAChQkKAGUOAY2RwM2AGZIJEAEYcOAa2bYA5mu1wK1bGSQAvLln5Dw0AFIcERWfGRoa4qUjhXBErimAoJAICAC4oe0cAbQBdWSjcfGIyCgAaWnpGLHSsE3MibOx2LjSdPQMsGKs-IgCoOUZgQggAE1lgggA6BrgmggAKfoA3LwBKDAA+KBGZVFhEVFGfARqUDgNOuA5tfqwevtaoBDQAWhPsJd9-dYhN7f795paloA

Reply:
Good catch, I fixed it and improved the typing so it's clear these aliases aren't going to be our `socialProviderLoginMechanisms`.